tests/test_scene_debugger.py

```diff
@@ -0,0 +1,107 @@
+from types import MappingProxyType
+
+import pytest
+from manim.animation.animation import Animation
+from manim.utils.debug.scene_debugger import SceneDebugger
+from manim.mobject.geometry import Square
+
+s = Square()
+a = Animation(s)
+dummy_scene_vars = MappingProxyType({"animations": [a], "mobjects": s, "dummy_attr": 1})
+dummy_renderer_vars = MappingProxyType(
+    {"time": 1, "number_frame": 2, "animations_hashes": ["999_999"], "num_plays": 0}
+)
+
+
+def test_scene_debugger_scene_infos():
+    s = SceneDebugger()
+    s.set_scene_vars(dummy_scene_vars)
+    s.set_renderer_vars(dummy_renderer_vars)
+    assert str(s._get_scene_dict_info()["animations"]) == "[Animation(Square)]"
+    assert s._get_scene_dict_info()["number_frame"] == 2
```

Comment:
I think this is testing the `t` logic and not so much the debug. This is okay, but will have to change once the frame starts at 0/1 instead of 2.